portality/constants.py

```diff
@@ -8,6 +8,7 @@
 APPLICATION_STATUS_COMPLETED = "completed"
 APPLICATION_STATUS_ON_HOLD = "on hold"
 APPLICATION_STATUS_READY = "ready"
+APPLICATION_STATUS_CREATED = "created"
```

Comment:
This isn't a new status.  I think the confusion here is that this notification has been wired to a "created" status, when in fact what we want is an event for the creation of a new application.  The initial application status will be "pending" or whatever it is set to.